hoomd/mpcd/ATCollisionMethod.cc

```diff
@@ -103,7 +104,7 @@ void mpcd::ATCollisionMethod::drawVelocities(unsigned int timestep)
             }
 
         // draw random velocities from normal distribution
-        hoomd::detail::Saru rng(tag, timestep, m_seed);
+        hoomd::detail::Saru rng(hoomd::RNGIdentifier::ATCollisionMethod, m_seed, tag, timestep);
         mpcd::detail::NormalGenerator<Scalar,true> gen;
```

Comment:
I am drawing normal variates here and elsewhere using a wrapper I wrote for MPCD, but I can update that separately to use the new Saru interface.